ush/rocoto/rocoto_viewer.py

```diff
@@ -194,9 +194,10 @@ def load_produtil_pythonpath():
     PRODUTIL = collections.defaultdict(list) 
     PRODUTIL['hera'] = '/scratch1/NCEPDEV/global/glopara/svn/nceplibs/produtil/trunk/ush'
     PRODUTIL['luna']  = '/gpfs/hps3/emc/global/noscrub/emc.glopara/svn/nceplibs/produtil/trunk/ush'
-    PRODUTIL['tide']  = '/gpfs/td1/emc/global/save/emc.glopara/svn/nceplibs/produtil/trunk/ush'
-    PRODUTIL['gyre']  = '/gpfs/gd1/emc/global/save/emc.glopara/svn/nceplibs/produtil/trunk/ush'
-    try_clusters = ('hera','luna','tide','gyre')
+    PRODUTIL['surge']  = '/gpfs/hps3/emc/global/noscrub/emc.glopara/svn/nceplibs/produtil/trunk/ush'
+    PRODUTIL['mars']  = '/gpfs/dell2/emc/modeling/noscrub/emc.glopara/svn/nceplibs/produtil/trunk/ush'
+    PRODUTIL['venus']  = '/gpfs/dell2/emc/modeling/noscrub/emc.glopara/svn/nceplibs/produtil/trunk/ush'
+    try_clusters = ('hera','luna','surge','mars','venus')
 
     for cluster in try_clusters:
```

Comment:
Loop on the `keys` of `PRODUTIL
```
for clusters in PRODUTIL.keys():
```

Reply:
Will ingest this suggestion in a separate issue, I have some other small edits to make. Thanks!